firebase-common/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Unreleased
 * [fixed] Correctly declare dependency on firebase-components, issue #5732
-
+* [changed] Added extension method `Random.nextAlphanumericString()` ([#5818](//github.com/firebase/firebase-android-sdk/pull/5818))
```

Comment:
Nit, but I'm curious about something:

@rachelsaunders in general, do we want to be more specific like this? Or is it preferable to be more vague and leave the pr link for specifics? eg, this versus say;
```
* [changed] Added an extension method for generating random alphanumeric strings #5818 
```

Or even

```
* [changed] Added a way to generate random alphanumeric strings #5818 
```

Reply:
This indicates that the method is public API. Is that the case @dconeybe ? if so we need api review for it.

Reply:
Ahh right. I didn't realize firebase-common was subject to api review. Do you think that this function is appropriate for firebase-common? It's really just an internal utility function that I want to use from 2 submodules of the product that I'm working on. I can just create a third, shared submodule for this class and just use a gradle "project" dependency.

Reply:
Is a bit complicated. `firebase-common` contains `FirebaseApp`, so we expose all public methods in it to developers (it should be declared as an `api` dependency in SDKs, but more on that later). Since this util method is public and publicly documented, then it becomes part of the public API, and then it needs to go through review. If it's not intended to be used by developers, and we don't document it publicly, bu using `@hide`, then AFAIK it doesn't need the review. 

Also, since common does not publish release notes, we don't need to be super accurate in our wording for the changelog.